source/_components/media_player.aquostv.markdown

```diff
@@ -27,14 +27,37 @@ media_player:
     host: 192.168.0.10
 ```
 
-Configuration variables:
-
-- **host** (*Required*): The IP/Hostname of the Sharp Aquos TV, eg. `192.168.0.10`.
-- **port** (*Optional*): The port of the Sharp Aquos TV. Defaults to 10002.
-- **username** (*Optional*): The username of the Sharp Aquos TV. Defaults to admin.
-- **password** (*Optional*): The password of the Sharp Aquos TV. Defaults to password.
-- **name** (*Optional*): The name you would like to give to the Sharp Aquos TV.
-- **power_on_enabled** (*Optional*): If you want to be able to turn on your TV. Defaults to False.
+{% configuration %}
+host:
+  description: The IP/Hostname of the Sharp Aquos TV, eg. `192.168.0.10`.
+  required: true
+  type: string
+port:
+  description: The port of the Sharp Aquos TV.
+  required: false
+  default: 10002
+  type: int
+username:
+  description: The username of the Sharp Aquos TV.
+  required: false
+  default: admin
+  type: string
+password:
+  description: The password of the Sharp Aquos TV.
+  required: false
+  default: password
+  type: string
+name:
+  description: The name you would like to give to the Sharp Aquos TV.
+  required: false
+  type: string
+power_on_enabled:
+  description: If you want to be able to turn on your TV.
+  required: false
+  default: false
+  type: string
```

Comment:
This should be `boolean`

Reply:
@dnguyen800 from that link:
```
type:: The type of the variable. Allowed entries: boolean, string, integer, float, time, template, device_class, icon or map/list (for a list of entries). For multiple possibilities use [string, integer]. If you use map/list then should define keys: (see the template sensor for an example). If you use boolean, then default: must be defined.
```
:)

Reply:
haha, that's funny! I am 100% sure it did not show boolean, float, device_class, or icon when I viewed that site yesterday. Anyways, I'll change it to boolean right now.